src/KurrentDB.SecondaryIndexing/Indexes/EventType/EventTypeIndexProcessor.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Kurrent, Inc and/or licensed to Kurrent, Inc under one or more agreements.
+// Kurrent, Inc licenses this file to you under the Kurrent License v1 (see LICENSE.md).
+
+using KurrentDB.Core.Bus;
+using KurrentDB.Core.Data;
+using KurrentDB.Core.Messages;
+using KurrentDB.SecondaryIndexing.Readers;
+using KurrentDB.SecondaryIndexing.Storage;
+using static KurrentDB.SecondaryIndexing.Indexes.EventType.EventTypeSql;
+
+namespace KurrentDB.SecondaryIndexing.Indexes.EventType;
+
+internal class EventTypeIndexProcessor {
+	private readonly Dictionary<string, int> _eventTypes;
```

Comment:
usually the number of event types should be relatively low, but there may be some users  who are using the system in an unconventional way. we could consider using an LRU cache here (or adding a metric to monitor the size of the dictionary)

Reply:
We don't really need a metric for that. We will have this information available for the user.